griffin-doc/measure/measure-configuration-guide.md

```diff
@@ -547,13 +547,15 @@ List of supported data connectors:
 | Name       | Type     | Description                            | Default Values |
 |:-----------|:---------|:---------------------------------------|:-------------- |
 | database   | `String` | database name                          | default |
+| sql        | `String` | query sql                              | `Empty` |
```

Comment:
is `sql` mode mutually exclusive to the usual table db mode or does it work with them? this needs to be documented somewhere.

Also how does this work now with the `where` functionality ?

Reply:
Thanks for the comment.  If the config `sql` was provided, `database`, `tablename` and `where` can all be ignored, connector will extract the result of sql. I added that description to the MD and added the SQL mode config example.